bazel/repositories.bzl

```diff
@@ -218,6 +219,16 @@ def _boringssl_fips():
         patches = ["@envoy//bazel/external:boringssl_fips.patch"],
     )
 
+def _com_github_cgetzen_cpp_infima():
```

Comment:
will this move to a public repo before commit?

Reply:
It is currently public here: https://github.com/cgetzen/cpp-infima
I'd like to see ownership transferred to github.com/envoyproxy, but there may be other options I haven't considered.

Reply:
I haven't reviewed the overall PR yet since it's still drafting, but just a heads up but we are unlikely to take a dependency on this external repo. I would just inline whatever code you actually need to make the PR function vs. using the library.

Reply:
Gotcha. This is a good opportunity for me to learn how licensing works.